### DISCUSSION TITLE
Install script documentation

### DISCUSSION THREAD

**Discussion by jacklowrie** [starship/starship#3787] [Q&A] [answered]
Is the install script documented anywhere? It looks like the script can take parameters based on other discussions, but I've been looking on the site docs and having a hard time locating.

**Comment by davidkna** [comment, marked as answer]
The shell script has a `-h`/`--help` option to list available parameters.

**Reply by jacklowrie** [reply]
Awesome, thanks! is there a way to be sure that no commands are executed with `sudo`? or alternatively, a manual install process? Tried that `-h` option to figure it out

**Reply by davidkna** [reply]
`sudo` should only be used if `bin-dir` is not writable. For a manual install, you can grab an appropriate build from the release page, extract it and put the binary into your `$PATH` or wherever you want it.
